ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/KsqlServerMainTest.java

```diff
@@ -90,6 +96,23 @@ public void shouldStopAppOnErrorStarting() throws Exception {
     verify(executable);
   }
 
+  @Test
+  public void shouldNotifyAppOnTerminate() throws Exception {
+    // Given:
+    final Capture<Runnable> captureShutdownHandler = newCapture();
+    shutdownHandler.execute(capture(captureShutdownHandler));
+    executable.notifyTerminated();
```

Comment:
I'm not understanding this test. Are we missing an `expectLastCall();` after this line?

Reply:
oof yeah - fixed